### PR TITLE
fix: Symbol upload test 2020

### PR DESCRIPTION
**Pull request by bitsandfoxes**
#skip-changelog

### CODE REVIEW

File: test/Scripts.Integration.Test/common.ps1 [hunk 1]
```diff
@@ -276,13 +278,15 @@ function CheckSymbolServerOutput([string] $buildMethod, [string] $symbolServerOu
             if ($unity2020OrHigher)
             {
                 $expectedFiles = @(
-                    "libil2cpp.sym.so: count=$($withSources ? 3 : 2)"
+                    "libil2cpp.sym.so: count=$($withSources ? 2 : 1)",
+                    "libil2cpp.dbg.so: count=$($withSources ? 3 : 2)"
```
Comment (vaind) [inline test/Scripts.Integration.Test/common.ps1:282]: won't this break 2021? THat worked before this change

Comment (bitsandfoxes) [reply test/Scripts.Integration.Test/common.ps1:282]: 2021 is completely special cased on line 251

Comment (vaind) [reply test/Scripts.Integration.Test/common.ps1:282]: right...

**Review by vaind — APPROVED** [review]
LGTM once the CI passes.

**Pull request merged**